docs/user/reference/debug/index.md

```diff
@@ -0,0 +1,39 @@
+# Debugging
+
+Different platforms have different ways to enable each debug functionality. They may be
+enabled
+
+- through APIs exposed on the Glean singleton,
+- through environment variables set at run time,
+- or through platform specific debug tools.
+
+## Platform Specific Information
+
+Check out the platform specific guides on how to use Glean's debug functionalities.
+
+1. [Debugging Android applications using the Glean SDK](../../user/debugging/android.md)
+2. [Debugging iOS applications using the Glean SDK](../../user/debugging/ios.md)
+3. [Debugging Python applications using the Glean SDK](../../user/debugging/python.md)
+4. [Debugging Javascript applications using Glean.js](../../user/debugging/javascript.md)
+
+## Features
+
+The Glean SDK provides four debugging features.
+
+### [Log Pings](./logPings.md)
```

Comment:
Links in headers: how discoverable are they? I know in the Firefox Source Docs they conflict with tables of contents and are really hard to discern since they are styled identically to unlinked headers.

Reply:
Links with headers turn blue in  `mdbook`. We use then quite a lot in this book https://mozilla.github.io/glean/book/index.html#sections.